content/articles/spotting-of-passing-objects-using-python/index.md

```diff
@@ -0,0 +1,165 @@
+﻿## Spotting of Passing Objects Using Python
+### Introduction
+Spotting of moving objects has become crucial in our lives today. Talking of security, cameras help us a great deal in executing this  
+function. Other key areas cut across tracking of living organisms,  vehicles in traffic control, and also replaceable where radar is discouraged.  
+We will be spotting objects in motion in this tutorial.  
+We will be requiring a few tools before carrying on, that is, the laptop's webcam and most of all an external library to be installed and downloaded which is the OpenCV module. With those at hand, we will be able to tackle our project. Let's  dive in.
```

Comment:
Add this under the pre-requisities section

Reply:
State the objective of this tutorial. What are we learning? Why? What is the code tutorial about?